bin/dev

```diff
@@ -81,7 +81,7 @@ runJstest(){
 # args: list of args to pass to pre-commit
 runPrecommit(){
     # if no cli args, run standard pre-push
-    if [ ! $* ]; then
+    if [ ! $1 ]; then
```

Comment:
`!` is an unary operator that takes in one argument. 
`$*` expands all the arguments
This causes bash to yell that you gave it too many arguments.
`! $1` simply retrieves the first argument and checks if it exists making bash happy.